__tests__/utxo.test.js

```diff
@@ -0,0 +1,26 @@
+import TestUtils from "./test-utils";
+
+describe("utxo api", () => {
+  it("utxo-filter should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .get("/wallet/utxo-filter")
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.utxos).toHaveLength(13);
+    expect(response.body.total_amount_available).toBe(83199);
+    expect(response.body.total_amount_locked).toBe(0);
+  });
+
+  it("utxo-consolidation should consolidate all utxos", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/utxo-consolidation")
+      .send({
+        destination_address: "WWbt2ww4W45YLUAumnumZiyWrABYDzCTdN",
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.total_utxos_consolidated).toBe(13);
+    expect(response.body.total_amount).toBe(83199);
+    expect(response.body.utxos).toHaveLength(13);
```

Comment:
I feel we should include another call to the `/wallet/utxo-filter` to double check the number of utxos has decreased. And also confirm that the wallet's balance is still the same.

Reply:
And double check the new utxo is in the correct destination address

Reply:
It requires mocking websocket messages coming from the `fullnode` that can be done later [here](https://github.com/HathorNetwork/hathor-wallet-headless/issues/88).